packages/material/scss/common/_forms.scss

```diff
@@ -56,6 +57,11 @@
             .k-select {
                 color: $icon-hovered-color;
             }
+
+            &::after {
+                transform: scale(1);
+                opacity: 1;
```

Comment:
What are these styles used for? It seems these are for the input ripple effects, which are achieved through the textbox container, and have no place in the theme.

Reply:
Those styles add ripple effect for all dropdowns and picker without a necessity of using javascript.